src/app/components/onc-organization/onc-organization-view.jsx

```diff
@@ -74,14 +66,38 @@ function ChplOncOrganizationView(props) {
              />
            </Typography>
          )}
-        { organization.address
+        </Box>
+        <Box className={classes.subContentBox}>
+          <Typography variant="body1" gutterBottom>
+            <strong>ONC-ACB code</strong>
+          </Typography>
+          <Typography>{ organization.acbCode}</Typography>
+        </Box>
+        <Box className={classes.subContentBox}>
+          <Typography variant="body1" gutterBottom>
+            <strong>Retired</strong>
+          </Typography>
+          { organization.retired ? 'Yes' : 'No' }
+        </Box>
+        <Box className={classes.subContentBox}>
+          {' '}
+          { organization.retired
```

Comment:
If "retired" is false, then this box only has a blank space in it, from line 84. This entire `Box` shouldn't exist if retired is false, so this line needs to be just above line 82